schema.sql

```diff
@@ -0,0 +1,23 @@
+-- Create table genre
+DROP TABLE IF EXISTS "genre";
+CREATE TABLE "genre"(
```

Comment:
It's better to join the commands together:

CREATE TABLE genre (
    id integer NOT NULL,
    name varchar(255) NOT NULL,
   
    PRIMARY KEY (id)
);

Reply:
Hi @richardoppiyo thanks for your comment. I update the query to include the constraint in the same statement:
CREATE TABLE genre(
    id INTEGER NOT NULL PRIMARY KEY,
    name VARCHAR(255) NOT NULL
);